layers/+lang/bibtex/packages.el

```diff
@@ -131,7 +131,10 @@
 
       "gj" 'ebib-jump-to-entry
       "/" 'ebib-search
-      "n" 'ebib-search-next)
+      "n" 'ebib-search-next
+      ;; the following binding is a simple workaround for
+      ;; https://github.com/joostkremers/ebib/issues/213
+      [remap spacemacs/kill-this-buffer] 'ebib-quit)
```

Comment:
I've not seen this syntax before. It turns out `evilified-state-evilify` uses `define-key`.